### PR TITLE
fix(design-tokens): fix scoped tokens in mfe

### PR DESCRIPTION
**Pull request by mkrds**
### Proposed behaviour

This PR fixes an issue where multiple `createGlobalStyle` calls override the already injected styles when multiple MFE's are loaded.

### Checklist

- [x] Commits follow our style guide
- [ ] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [x] Unit tests added or updated if required
- [ ] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Testing instructions

The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.

### CODE REVIEW

File: src/style/design-tokens/carbon-scoped-tokens-provider/carbon-scoped-tokens-provider.component.js [hunk 1]
```diff
@@ -21,32 +21,25 @@ const kebabCase = (input) =>
...
   display: inline;
+
+  ${({ theme }) => generateCssVariables(theme.compatibility)}
 `;
 
+TokensProviderWrapper.defaultProps = {
+  theme: baseTheme,
+};
+
 const CarbonScopedTokensProvider = ({ children }) => (
```
Comment (jamime) [inline src/style/design-tokens/carbon-scoped-tokens-provider/carbon-scoped-tokens-provider.component.js:37]: I think we could delete `TokensProviderWrapper` and add the styles to `CarbonScopedTokensProvider` to reduce the number of components.

**Review by jamime — CHANGES_REQUESTED** [review]

File: src/style/design-tokens/carbon-scoped-tokens-provider/carbon-scoped-tokens-provider.component.js [hunk 2]
```diff
@@ -21,32 +21,25 @@ const kebabCase = (input) =>
 export const tokensClassName = (theme) =>
```
Comment (samtjo) [inline src/style/design-tokens/carbon-scoped-tokens-provider/carbon-scoped-tokens-provider.component.js:21]: Is there any need to keep this anymore now that we're not adding the CSS variables to elements with this class?

**Review by samtjo — COMMENTED** [review]
`tokensClassName` is also used in Portal and Tooltip - do you need to change the implementations in those components as well?

**Review by samtjo — APPROVED** [review]

**Review by jamime — APPROVED** [review]

**Review by samtjo — APPROVED** [review]

**Pull request merged**